pkgs/tools/video/nordlicht/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, cmake, ffmpeg, popt, help2man }:
+    with cmake;
+    stdenv.mkDerivation rec {
+        name = "nordlicht-${version}";
+        version = "0.4.5-git.7ad3f008.4b1751bd";
+        license = "GNU GPL v2 or later";
```

Comment:
goes into meta section

Reply:
... and should be `license = stdenv.lib.licenses.gpl2Plus`